@here/harp-mapview-decoder/lib/TileDataSource.ts

```diff
@@ -258,6 +262,17 @@ export class TileDataSource<TileType extends Tile> extends DataSource {
             );
             tile.tileLoader = newTileLoader;
             tile.copyrightInfo = this.m_options.copyrightInfo;
+            if (this.m_options.copyrightProvider !== undefined) {
+                this.m_options.copyrightProvider
+                    .getCopyrights(tile.geoBox, tileKey.level)
+                    .then(copyrightInfo => {
+                        tile.copyrightInfo =
+                            tile.copyrightInfo === undefined
+                                ? copyrightInfo
+                                : [...tile.copyrightInfo, ...copyrightInfo];
```

Comment:
Please add this behavior to the API documentation above (i.e. that copyrightInfo and copyright provider copyrights are concatenated)

Reply:
Updated typedoc to explain this behavior.